.venv/bin/getArch.py

```
@@ -0,0 +1,117 @@
+                stringBinding = r'ncacn_ip_tcp:%s[135]' % machine
+                transport = DCERPCTransportFactory(stringBinding)
+                transport.set_connect_timeout(int(self.__options.timeout))
+                dce = transport.get_dce_rpc()
+                dce.connect()
+                try:
+                    dce.bind(MSRPC_UUID_PORTMAP, transfer_syntax=self.NDR64Syntax)
+                except DCERPCException as e:
+                    if str(e).find('syntaxes_not_supported') >= 0:
+                        print('%s is 32-bit' % machine)
+                    else:
+                        logging.error(str(e))
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)